LoggingKata/TacoParser.cs

```diff
@@ -6,13 +6,37 @@
     public class TacoParser
     {
         readonly ILog logger = new TacoLogger();
-        
+
         public ITrackable Parse(string line)
         {
+            if(line == null)
+            {
+                return null;
+            }
+            var cells = line.Split(',');
+            if (cells.Length < 3 || cells.Length > 3)
+            {
+                return null;
+            }
             logger.LogInfo("Begin parsing");
-
-            // Do not fail if one record parsing fails, return null
-            return null; // TODO Implement
+            TacoBell newTacoBell = new TacoBell();
+            double Latitude;
+            double Longitude;
+            string string_Latitude = cells[0];
```

Comment:
Naming convention-wise, C# tends to use pascal or camel case, with Python usually using snake case. Consider naming these with camel as `stringLatitude` or Hungarian notation as `strLatitude`